docs/configuration/install.md

```diff
@@ -192,6 +192,11 @@ Usually Apache ships with a default configuration (e.g. ```/etc/apache2/sites-en
         AllowOverride All
         Require all granted
     </Directory>
+    <Directory /var/drbd/www/html/ilias/setup/>
+        Order Deny,Allow
+        Deny from all
+        Allow from 127
```

Comment:
Like @gpkvt already mentioned, the reference system uses Apache 2.4. The syntax for access rules is somewhat different in this version. Also, I personally find it better to use subnet notation, even though your example might work too. You might want to reduce the whole block to this statement:
```
Require ip 127.0.0.0/8
```

Reply:
I removed the complete block and referred to the security-section instead.